src/reducers/user-reducer.ts

```diff
@@ -0,0 +1,59 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import type { AppDispatch } from "../store";
+import { createNewUser } from "../requests";
+
+interface User {
+	userId: string;
+	trackingInterval: number;
+	sendingInterval: number;
+}
+
+const initialState: User = {
+	userId: null,
```

Comment:
See the `route-reducer` comment on strict null checks!

Reply:
Done